dev-libs/OpenNI2/OpenNI2-9999.ebuild

```diff
@@ -32,12 +33,12 @@ RDEPEND="
 	virtual/libudev
 	virtual/jpeg:0
 	opengl? ( media-libs/freeglut )
-	java? ( >=virtual/jre-1.5:* )
+	java? ( >=virtual/jre-1.5:= )
 "
 DEPEND="${RDEPEND}
-	dev-lang/python
+	${PYTHON_DEPS}
 	doc? ( app-doc/doxygen )
-	java? ( >=virtual/jdk-1.5:* )"
```

Comment:
Reinstated, suspected it would be covered by DEPEND=${RDEPEND} but who am I to question your logic :D